packages/stop-viewer-overlay/src/index.js

```diff
@@ -0,0 +1,71 @@
+import PropTypes from "prop-types";
+import React from "react";
+import {
+  FeatureGroup,
+  MapLayer,
+  Popup,
+  CircleMarker,
+  withLeaflet
+} from "react-leaflet";
+
+class StopViewerOverlay extends MapLayer {
```

Comment:
I agree with @fpurcell's comment about usage of this component. It would be good to add JSdoc regarding how this works.